src/data/reasons.js

```diff
@@ -53,6 +53,11 @@ export default {
             "url": "https://www.pcgamer.com/microsoft-will-soon-force-internet-explorer-users-to-open-edge-for-some-websites/",
             "label": "Microsoft forces usage of Edge instead of IE11",
             "description": "by Paul Lilly"
+        },
+        {
+            "url": "http://whyiesucks.blogspot.com/",
+            "label": "Why IE Sucks",
+            "description": "by http://whyiesucks.blogspot.com/"
```

Comment:
If the author is unknown, we can leave out this attribute

Reply:
ok fixed